### PR TITLE
feat: sdk update

### PR DESCRIPTION
**Pull request by avniculae**

### CODE REVIEW

File: yalc.lock [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "version": "v1",
   "packages": {
     "@voltz-protocol/v1-sdk": {
-      "signature": "a463c83a8b9b4cb0d4e3090ddea9682d",
+      "signature": "ed66f131a2a53649977506cb7b60901e",
```
Comment (ssbarbee) [inline yalc.lock:5]: is this what is live on "main" branch atm?

**Comment by arturbeg** [comment]
hey @avniculae there are quite a few conflicts now, worth resolving them with the help of @ssbarbee

**Comment by arturbeg** [comment]
also will be helpful @avniculae if you can provide a detailed description of the PR highlighting the changes that were introduced, motivation behind these changes + if relevant links to discord threads, etc

**Review by arturbeg — COMMENTED** [review]
why do we have so many changes to typechain?

**Pull request closed without merging**